std/assembly/util/string.ts

```diff
@@ -85,13 +88,19 @@ export function strtol<T>(str: string, radix: i32 = 0): T {
   }
   if (code == CharCode.MINUS) {
     // @ts-ignore: cast
-    if (!--len) return <T>NaN;
+    if (!--len) {
+      if (isFloat<T>()) return <T>NaN;
```

Comment:
Does the ts-ignore above still catch this? Might be that it now requires to move the ignore to this line and add one to the next.

Reply:
yeah. Fixed